pint/models/dispersion_model.py

```diff
@@ -65,16 +105,49 @@ def dispersion_delay(self, toas):
 
         return self.dispersion_time_delay(dm, bfreq)
 
-    def d_delay_d_DM(self, toas):
+    def print_par_DMs(self,):
+        # TODO we need to have a better design for print out the parameters in
+        # an inhertance class.  
+        result  = ''
+        perfix_dm = self.get_prefix_mapping('DM').values()
```

Comment:
Fix the spelling errror ("perfix" to "prefix") and add list() to convert the values to a list for Python3.